core/templates/dev/head/pages/collection_editor/collection_editor.html

```diff
@@ -117,6 +117,7 @@
 
   {# TODO(bhenning): Remove these once RouterServices is decoupled from the exploration editor #}
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/EditorServices.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/ExplorationWarningsService.js"></script>
```

Comment:
@shubha1593, Is it required?
I didn't find any services from exploration_editor/EditorServices.js which is used in collection_editor.

Reply:
@DubeySandeep, `ExplorationWarningsService` is a dependency for `ExplorationEditor` controller which is defined in pages/exploration_editor/ExplorationEditor.js and that is being imported in collection_editor.html But the thing is this controller is never used anywhere except in exploration_editor.html !(Which means we don't even need ExplorationEditior.js in collection_editor).
So yup we don't need to import `ExplorationWarningsService` here.